modules/delegates.js

```diff
@@ -642,24 +532,22 @@ __private.loadDelegates = function(cb) {
 					if (!account) {
 						return setImmediate(
 							seriesCb,
-							[
-								'Account with public key:',
-								keypair.publicKey.toString('hex'),
-								'not found',
-							].join(' ')
+							['Account with public key:', keypair.publicKey, 'not found'].join(
+								' '
+							)
 						);
 					}
 
 					if (account.isDelegate) {
-						__private.keypairs[keypair.publicKey.toString('hex')] = keypair;
+						__private.keypairs[keypair.publicKey] = keypair;
```

Comment:
So `keypair.publicKey`mand `keypair.privateKey` are both strings now?

It will not fail here?
https://github.com/LiskHQ/lisk/blob/0daca9dd47f026a9344516a5e3c9b2b74c4fbe70/logic/block.js#L137

Reply:
Yes both are hex strings now. I can look into occurrences to fix it.